pkgs/by-name/co/comfyui/custom-nodes.nix

```diff
@@ -0,0 +1,32 @@
+{
+  fetchFromGitHub
+, stdenv
+, lib
+}:
+
+let
+  mkComfyUICustomNodes = args: stdenv.mkDerivation ({
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/
+      cp -r $src/* $out/
+      runHook postInstall
+    '';
+
+    passthru.dependencies = [];
+  } // args);
+in
+{
+  ultimate-sd-upscale = mkComfyUICustomNodes {
+    pname = "ultimate-sd-upscale";
+    version = "unstable-2023-08-16";
+
+    src = fetchFromGitHub {
+      owner = "ssitu";
+      repo = "ComfyUI_UltimateSDUpscale";
+      rev = "6ea48202a76ccf5904ddfa85f826efa80dd50520";
+      hash = "sha256-fUZ0AO+LARa+x30Iu+8jvEDun4T3f9G0DOlB2XNxY9Q=";
+      fetchSubmodules = true;
+    };
```

Comment:
Can we move this to it's own package?

Reply:
I might need some help to figure out where I can define the `mkConfyUICustomNodes` function so that I can then import it from the `.nix` file that defines each custom node.

Reply:
I think I misread the code here on my first review pass.
This is probably fine as it is, and we can safely ignore my comment here.
May want to consider patching shebangs with ``patchShebangs`` on the output however.
Just if you see anything particularly bad like ``#!/bin/bash``, etc.